src/main/java/edu/kis/vh/nursery/containers/IntArrayStack.java

```diff
@@ -0,0 +1,64 @@
+package edu.kis.vh.nursery.containers;
+
+import edu.kis.vh.nursery.IntStack;
+
+public class IntArrayStack implements IntStack {
+	private static final int STACK_SIZE = 12;
+	private static final int FULL_COUNT = STACK_SIZE;
+	private int[] numbers = new int[STACK_SIZE];
+
+	private int total = IntStack.EMPTY;
+
+	/* (non-Javadoc)
+	 * @see edu.kis.vh.nursery.IntStack#getTotal()
+	 */
+	@Override
+	public int getTotal() {
+		return total;
+	}
+
+	/* (non-Javadoc)
+	 * @see edu.kis.vh.nursery.IntStack#countIn(int)
+	 */
+	@Override
+	public void countIn(int in) {
+		if (!isFull())
+			numbers[total++] = in;
```

Comment:
Changing EMPTY value requires such fixes? It does not make sense. Incorrect fix.